libminifi/CMakeLists.txt

```diff
@@ -48,6 +49,19 @@ file(GLOB SPD_SOURCES "../include/spdlog/*")
 add_library(spdlog STATIC ${SPD_SOURCES})
 add_library(minifi STATIC ${SOURCES})
 
+# Include Boost System
+find_package(Boost COMPONENTS system REQUIRED)
+target_link_libraries(minifi ${Boost_SYSTEM_LIBRARY})
+
+# Include libxml2
```

Comment:
Okay, may have just been an artifact.  This was removed as per MINIFI-187 (https://github.com/apache/nifi-minifi-cpp/commit/0b022245d46371567077efd128cbf21a1657aa4d)